test/e2e/step_definitions/generic.step.js

```diff
@@ -291,6 +291,14 @@ defineSupportCode(({ Given, When, Then, setDefaultTimeout }) => {
     optionElem.click().then(callback).catch(callback);
   });
 
+  Then('I click on "{className}" element no. {index}', (className, index, callback) => {
+    browser.sleep(500);
+    const optionElem = element.all(by.css(className)).get(index - 1);
+    browser.wait(EC.presenceOf(optionElem), waitTime)
+      .catch(error => console.error(`${error}`)); // eslint-disable-line no-console
+    optionElem.click().then(callback).catch(callback);
```

Comment:
`optionElem.click()` should be IMO in `then` of `browser.wait` because it's async.